tests/template_backends/test_django.py

```diff
@@ -145,3 +146,32 @@ def test_non_debug_default_template_loaders(self):
     def test_debug_default_template_loaders(self):
         engine = DjangoTemplates({'DIRS': [], 'APP_DIRS': True, 'NAME': 'django', 'OPTIONS': {}})
         self.assertEqual(engine.engine.loaders, self.default_loaders)
+
+    @override_settings(DEBUG=True)
+    def test_debug_template_loaders_with_dirs(self):
+        engine = DjangoTemplates({
+            'DIRS': ['dirs'],
+            'APP_DIRS': True,
+            'NAME': 'django',
+            'OPTIONS': {},
+            'POST_APP_DIRS': ['post_app_dirs'],
+        })
```

Comment:
I see, thanks for your answer. I really don't want to hold the template based widget stuff from landing any longer. I suppose this is something we could refactor later on.